TimePlatformer/Assets/Scripts/PauseMenuScript.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PauseMenuScript : MonoBehaviour {
+
+
+    public void pause(){
+        Debug.Log("pausing");
+        Time.timeScale = 0;
+        var continueButton = GameObject.FindGameObjectWithTag("ContinueButton");
+        var restartButton = GameObject.FindGameObjectWithTag("RestartButton");
+        var mainMenuButton = GameObject.FindGameObjectWithTag("MainMenuButton");
+        var exitButton = GameObject.FindGameObjectWithTag("ExitGameButton");
+        var pauseButton = GameObject.FindGameObjectWithTag("PauseButton");
+
+
+        continueButton.transform.position = new Vector3(850.0f, 500.0f);
+        restartButton.transform.position = new Vector3(850.0f, 440.0f);
+        mainMenuButton.transform.position = new Vector3(850.0f, 380.0f);
+        exitButton.transform.position = new Vector3(850.0f, 320.0f);
+        pauseButton.transform.position = new Vector3(-10000.0f, 120.0f);
+    }                  
+
+
+    // If continue button is clicked it will unpause the level.
+    // It will enable pause button render and disable the other buttons as well as itself
+    // resume animation
+    public void continueGame(){
+        Time.timeScale = 1;
+        var continueButton = GameObject.FindGameObjectWithTag("ContinueButton");
```

Comment:
Are you just moving the buttons off screen? Why not actually enable or disable their gameobjects?